src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -147,7 +149,7 @@ def get(self, request: Request, group) -> Response:
             return Response([])  # No exception, stacktrace or in-app frames
 
         similar_issues_params: SimilarIssuesEmbeddingsRequest = {
-            "hash": latest_event.get_primary_hash(),
+            "hash": NonNone(latest_event.get_primary_hash()),
```

Comment:
please remove this helper -- this is going to cause a lot of pain to undo.  please don't introduce things like this without a review from the typing owners